compiler/rustc_target/src/spec/x86_64_unknown_none.rs

```diff
@@ -0,0 +1,41 @@
+// Generic x86-64 target for bare-metal code - Floating point disabled
+//
+// Can be used in conjunction with the `target-feature` and
+// `target-cpu` compiler flags to opt-in more hardware-specific
+// features.
+
+use super::{
+    CodeModel, LinkerFlavor, LldFlavor, PanicStrategy, RelocModel, RelroLevel, StackProbeType,
+    Target, TargetOptions,
+};
+
+pub fn target() -> Target {
+    let opts = TargetOptions {
+        cpu: "x86-64".to_string(),
+        max_atomic_width: Some(64),
+        // don't use probe-stack=inline-asm until rust#83139 and rust#84667 are resolved
+        stack_probes: StackProbeType::Call,
+        position_independent_executables: true,
+        static_position_independent_executables: true,
+        relro_level: RelroLevel::Full,
+        relocation_model: RelocModel::Static,
```

Comment:
@haraldh We are currently using `relocation-model=pic` in our code. But I don't remember why. Does this represent a problem for us?

Reply:
I thought that we had changed this; I absolutely think we should set pic as the default.

Reply:
Fixed in https://github.com/mikeleany/rust/pull/3

Reply:
> @haraldh We are currently using `relocation-model=pic` in our code. But I don't remember why. Does this represent a problem for us?

@npmccallum thx for spotting this